LondonBritishLibrary/orient/BLorient13272.xml

```diff
@@ -0,0 +1,283 @@
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">178</measure>
+                                    <measure unit="page" type="blank" corresp="#1v #2r #2v #178v">4</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>175</height>
+                                        <width>120</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="19 20"><locus from="3r" to="160r"></locus>
+                                </layout>
+                                <layout columns="2" writtenLines="19 20"><locus from="160v" to="178r"/></layout>
+                            </layoutDesc>
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Rather careful handwriting.</desc>
+                                <date notBefore="1800" notAfter="1978" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="band">
+                                <desc>Bands of coloured geometric ornamentation.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d2" type="drawing">
+                                <locus target="#160r"></locus>
+                                <desc>Traces of a drawing representing heads.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="176ra" to="178ra"/>
+                                    <desc type="GuestText"><ref type="work" corresp="LIT3108RepCh388">Praise of the Virgin Mary</ref>.</desc>
+                                    <q xml:lang="gez">ይዌድስዋ፡ መላእክት፡ ለማርያም፡ በውስተ፡ ውሳጤ፡ መንጦላእክት፡ ወይቤልዋ፡ በሐኪ፡ ማርያም፡ </q>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus target="#178ra"/>
+                                    <desc type="GuestText">Magical prayer against sores.</desc>
+                                    <q xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎት፡ በእንተ፡ ቍስል፡ <gap reason="ellipsis"/></q>
```

Comment:
same as ESmr022 f. 81r?

Reply:
This is possible, but I cannot say with certainty, because the four words quoted by Strelcyn is everything, what I have. I will add this possible attestation in the desc to a2.